src/breakpoint/DisableHWBreakpoint.ts

```diff
@@ -0,0 +1,34 @@
+import { ExtensionContext } from 'vscode';
+import * as vscode from 'vscode';
+
+export class DisableHWBreakpoint {
+    constructor(context: ExtensionContext) {
+        this.registerCommands(context);
+    }
+
+    private readonly registerCommands = (context: ExtensionContext) => {
+        context.subscriptions.push(
+            vscode.commands.registerCommand(
+                'cdt.debug.disableHWBreakpoint',
+                async () => {
+                    const session = vscode.debug.activeDebugSession;
+                    if (session) {
+                        await vscode.commands.executeCommand(
```

Comment:
Can you simplify this to a single context?

Reply:
The solution has been changed to use `Toggle Breakpoint`